sos/plugins/openswan.py

```diff
@@ -42,6 +42,8 @@ def setup(self):
         if self.get_option("ipsec-barf"):
             self.add_cmd_output("ipsec barf")
 
-        self.add_forbidden_path("/etc/ipsec.d/key[3-4].db")
+        self.add_forbidden_path('/etc/ipsec.secrets')
+        self.add_forbidden_path('/etc/ipsec.d/*.db')
+        self.add_forbidden_path('/etc/ipsec.d/*.secrets')
```

Comment:
You may also want to forbid /etc/ipsec.secrets.d/* for older installations.

Reply:
Done, thanks!

Reply:
These days the `add_forbidden_path()` interface accepts lists as well as strings (like `add_copy_spec()` and `add_cmd_output()`, so you can replace these with a single call:

```python
        self.add_forbidden_path([
            '/etc/ipsec.secrets',
            '/etc/ipsec.d/*.db',
            '/etc/ipsec.d/*.secrets'
        ])
```

Reply:
:+1: TIL; done, thanks!

Reply:
Thanks!